sap-s4hana-cloud/2.1/modules/ROOT/pages/sap-s4hana-cloud-connector-upgrade-migrate.adoc

```diff
@@ -66,7 +65,7 @@ If there are problems with caching the parameters and metadata, try restarting S
 
 == Revert the Upgrade
 
-If it is necessary to revert to the previous version of SAP S/4HANA OData Cloud Connector, change the `mule-sap-s4hana-cloud-connector` dependency version 2.0.0 in the project’s `pom.xml` to the version to which you want to revert.
+If it is necessary to revert to the previous version of SAP S/4HANA OData Cloud Connector, change the `mule-sap-s4hana-cloud-connector` dependency version to the previous version in the project’s `pom.xml` to the version to which you want to revert.
```

Comment:
Please check if the connector's name should be w/o 'Cloud' here, too. @valkyrie69